src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php

```diff
@@ -11,20 +11,20 @@
 
 namespace Sulu\Component\Rest\Tests\Unit\ListBuilder;
 
-use Doctrine\Common\Persistence\ObjectManager;
 use Sulu\Component\Rest\ListBuilder\ListRestHelper;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 class ListRestHelperTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @var ObjectManager
+     * @var RequestStack|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
We usually don't do that. Does PHPStorm then really recognize that methods of both classes are available? However, it would not be necessary if we switch to prophecy.